GOVERNANCE.md

```diff
@@ -0,0 +1,149 @@
+# Rez Project Roles and Responsibilities
+
+Rez is a project of the Academy Software Foundation and relies on
+the ASWF governance policies, supported by the Linux Foundation.
+
+There are three primary project roles: 
+
+* [Contributors](#Contributors) submit code to the project
+* [Committers](#Committers) approve code to be included into the project
+* [Technical Steering Committee (TSC)](#Technical-Steering-Committee) 
```

Comment:
Open question to rest of TSC:
Do we want "TSC" and "rez-maintainers" to be synonymous in all ways? If not, it may be worth separating those two concepts here. If so, it may still be worth mentioning here that the TSC members are part of the "@AcademySoftwareFoundation/rez-maintainers" group.

Reply:
We could ask to rename group to `rez-tsc`. Not only it's shorter, but it's also clearer who is part of that group since the is no mention of "maintainer" in the governance model.

Reply:
it's a good question. I could see eventually that we may have more maintainers than TSC members. But I like the idea of a rez-tsc group (as there is no ambiguity). I'd say we rename it now, we can always add a new rez-maintainers group later on.

Reply:
It's possible that the Committer/Maintainer role comprises several different permission roles -- reviewers, merge access, code owners, some admin permissions. I don't think it's important for the governance document to clarify all the specific GitHub permissions or that we need to figure that all out now. The important part is acknowledging the opportunity for contributors to take on more responsibility within the project.

Reply:
Allan will rename the group to rez-tsc so this verbage is fine.